org.eclipse.buildship.core/src/main/resources/org/eclipse/buildship/core/i18n/CoreMessages.properties

```diff
@@ -49,4 +49,5 @@ ErrorMessage_0_MustBeDirectory={0} must be a directory.
 ErrorMessage_0_AlreadyExists={0} already exists.
 
 Preference_Label_GradleUserHome=Gradle User Home
-Preference_Label_OfflineMode=Offline mode
\ No newline at end of file
+Preference_Label_OfflineMode=Offline mode
+Preference_Label_BuildScans=Enable Build Scans
```

Comment:
I agree with this proposal, although it would be really nice to find a consistent solution how to show contextual help for any functionality. A popup bubble, maybe?